examples/web/index.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="UTF-8">
   <meta content="IE=Edge" http-equiv="X-UA-Compatible">
   <meta name="description" content="A set of small game examples showcasing each feature provided by the Flame Engine">
+  <base href="$FLUTTER_BASE_HREF">
```

Comment:
We also need to add the baseHref arg on the action right? Or maybe not because we already using the root?

Reply:
I added `https://examples.flame-engine.org/` first, but then it is not going to work if you are watching the deploy on github.
The default is `/`, and we should probably be fine with that right? Arguable we wouldn't even need the `base` tag since we don't have any hierarchical structure on the site.

Reply:
Yeah, makes sense the default is already the root.